quodlibet/quodlibet/ext/events/qlscrobbler.py

```diff
@@ -313,12 +313,14 @@ def send_submission(self):
         to_submit = self.queue[:min(len(self.queue), 50)]
         for idx, song in enumerate(to_submit):
             for key, val in song.items():
-                data['%s[%d]' % (key, idx)] = val.encode('utf-8')
+                data['%s[%d]' % (key, idx)] = val
             data['o[%d]' % idx] = 'P'
             data['r[%d]' % idx] = ''
 
         print_d('Submitting song(s): %s' %
-            ('\n\t'.join(['%s - %s' % (s['a'], s['t']) for s in to_submit])))
+            ('\n\t'.join(['%s - %s' % (
+                s[b'a' if b'a' in s else 'a'],
```

Comment:
that looks hacky, why are there mixed types here?

Reply:
I have no idea. I dumped my 'to_submit' and it had both 'a' and b'a' for some reason.

Reply:
It might have been a part of unsubmitted songs that I played while plugin wasn't working but I'm unsure about that.